src/main/java/cloud/prefab/client/ConfigClient.java

```diff
@@ -150,11 +169,10 @@ public boolean removeConfigChangeListener(ConfigChangeListener configChangeListe
     return configChangeListeners.remove(configChangeListener);
   }
 
-  @Override
   public Collection<String> getKeys() {
     try {
       initializedLatch.await();
```

Comment:
should this have the same await timeout and unitialized handling as the `get` method above?

Reply:
better yet let's remove it. This was here because this class was implementing the older ConfigStore interface. Since we've changed that, I don't know the utility of it.